kzz7yh-a10381/cod-ve07v1_kzz7yh-a10381.xml

```diff
@@ -241,8 +245,8 @@
       pre<lb ed="#V" n="110" break="no"/>supponit aliquam de deo cognitionem. 
     </p>
                <p xml:id="kzz7yh-a10381-d1e447">
-                  <lb ed="#V" n="111"/>Contra. Auga eququis c eaigens de de  
-      deri<lb ed="#V" n="112" break="no"/>sectus: quia anima no eot acquirere cognitionem 
+                  <lb ed="#V" n="111"/>Contra Nostra prima nationalis cognitio de deo est per 
+      ef<lb ed="#V" n="112" break="no"/>fectus: quia anima no potest acquerere cognitionem 
```

Comment:
q with the line through the stem is usually an abbreviation for "qui" as we see here in "acquirere"

```suggestion
      ef<lb ed="#V" n="112" break="no"/>fectus: quia anima non potest acquirere cognitionem 
```